lib/stimulus_reflex/broadcasters/selector_broadcaster/update.rb

```diff
@@ -0,0 +1,19 @@
+class Update
+  include CableReady::Identifiable
+
+  def initialize(key, value, reflex)
+    @key = key
+    @value = value
+    @reflex = reflex
+  end
+
+  def selector
+    @selector ||= identifiable?(@key) ? dom_id(@key) : @key.to_s
+  end
+
+  def html
+    html = @reflex.render(@key) if @key.is_a?(ActiveRecord::Base) && @value.nil?
+    html = @reflex.render_collection(@key) if @key.is_a?(ActiveRecord::Relation) && @value.nil?
+    html || @value
+  end
+end
```

Comment:
When I noted that we could drop the namespace, it was in the context of defining a struct inside the proper scope. Now that this is an actual class definition we should properly namespace it as `StimulusReflex::SelectorBroadcaster::Update`. Sorry for the confusion.

Reply:
of course...